sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -824,32 +824,32 @@ public Mono<ModelOperation> getOperation(String operationId) {
      * String operationId = &quot;&#123;operation_Id&#125;&quot;;
      * documentModelAdministrationAsyncClient.getOperationWithResponse&#40;operationId&#41;.subscribe&#40;response -&gt; &#123;
      *     System.out.printf&#40;&quot;Response Status Code: %d.&quot;, response.getStatusCode&#40;&#41;&#41;;
-     *     ModelOperation modelOperation = response.getValue&#40;&#41;;
-     *     System.out.printf&#40;&quot;Operation ID: %s%n&quot;, modelOperation.getOperationId&#40;&#41;&#41;;
-     *     System.out.printf&#40;&quot;Operation Kind: %s%n&quot;, modelOperation.getKind&#40;&#41;&#41;;
-     *     System.out.printf&#40;&quot;Operation Status: %s%n&quot;, modelOperation.getStatus&#40;&#41;&#41;;
-     *     System.out.printf&#40;&quot;Model ID created with this operation: %s%n&quot;, modelOperation.getModelId&#40;&#41;&#41;;
-     *     if &#40;ModelOperationStatus.FAILED.equals&#40;modelOperation.getStatus&#40;&#41;&#41;&#41; &#123;
-     *         System.out.printf&#40;&quot;Operation fail error: %s%n&quot;, modelOperation.getError&#40;&#41;.getMessage&#40;&#41;&#41;;
+     *     DocumentModelOperationInfo modelOperationInfo = response.getValue&#40;&#41;;
+     *     System.out.printf&#40;&quot;Operation ID: %s%n&quot;, modelOperationInfo.getOperationId&#40;&#41;&#41;;
+     *     System.out.printf&#40;&quot;Operation Kind: %s%n&quot;, modelOperationInfo.getKind&#40;&#41;&#41;;
+     *     System.out.printf&#40;&quot;Operation Status: %s%n&quot;, modelOperationInfo.getStatus&#40;&#41;&#41;;
+     *     System.out.printf&#40;&quot;Model ID created with this operation: %s%n&quot;, modelOperationInfo.getModelId&#40;&#41;&#41;;
+     *     if &#40;ModelOperationStatus.FAILED.equals&#40;modelOperationInfo.getStatus&#40;&#41;&#41;&#41; &#123;
+     *         System.out.printf&#40;&quot;Operation fail error: %s%n&quot;, modelOperationInfo.getError&#40;&#41;.getMessage&#40;&#41;&#41;;
      *     &#125;
      * &#125;&#41;;
      * </pre>
      * <!-- end com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.getOperationWithResponse#string -->
      *
      * @param operationId Unique operation ID.
-     * @return A {@link Response} containing the requested {@link ModelOperation}.
+     * @return A {@link Response} containing the requested {@link DocumentModelOperationInfo}.
```

Comment:
From our discussion with Paul it wasnt clear to me that ModelOperation/ModelOperationInfo could follow the same pattern as DocumentModelInfo/DocumentModelSummary. Did Paul agree to that in the end?

Reply:
I thought we wanted to keep this naming regardless of if we could get that model pattern in the swagger.

@witemple-msft @HarshaNalluru @kinelski